docs/en/week07/07-3.md

```diff
@@ -0,0 +1,143 @@
+---
+lang-ref: ch.07-3
+title: Introduction to autoencoders
+lecturer: Alfredo Canziani
+authors: Vidit Bhargava, Monika Dagar
+date: 18 March 2021
+---
+$\newcommand{\vert}{\boldsymbol}$
+$\newcommand{\matr}{\boldsymbol}$
+## Applications of Autoencoder
+### DALL-E: Creating Images from Text
+
+DALL-E (released by OpenAL) is a neural network based on transformers architecture that creates images from text captions. It is a 12-billion parameter version of GPT-3, trained on a dataset of text-image pairs.
+
+<center>
+<img src="{{site.baseurl}}/images/week07/07-3/DALL-E.png" style="background-color:#DCDCDC;" /><br>
+<b>Figure 1:</b> DALL-E: Input-Output
+</center>
+
+Go to the [website](https://openai.com/blog/dall-e/) and play with the captions! 
+
+## Autoencoder
+Let's start with some definitions:
+
+### Definitions
+#### Input
+$\vert{x}$: is observed during both training and testing 
+
+$\vert{y}$: is observed during training but not testing
+
+$\vert{z}$: is not observed (neither during training nor during testing).
+
+#### Output
+$\vert{h}$: is computed from the input (hidden/internal)
+
+$\vert{\tilde{y}}$: is computed from the hidden (predicted $\vert{y}$, ~ means *circa*)
```

Comment:
![image](https://user-images.githubusercontent.com/2119355/119248664-b736ab80-bb60-11eb-8fb9-5229c78befd5.png)

Did you actually run the website locally?

Reply:
Just use `\vect{}` instead of the absolute value `\vert`.